http/src/main/resources/reference.conf

```diff
@@ -25,7 +25,7 @@ dataproc {
   legacyCustomDataprocImage = "projects/broad-dsp-gcr-public/global/images/custom-leo-image-1-2-79-debian9-251b76c"
   customDataprocImage = "projects/broad-dsp-gcr-public/global/images/custom-leo-image-1-4-15-debian9-251b76c"
 
-  dataprocReservedMemory = 4.5g
+  dataprocReservedMemory = 5g
```

Comment:
Unfortunately `4.5g` was a bit close to the limit -- I see better results when reserving 5g on standard and highmem machine types.